src/index.html

```diff
@@ -22,6 +22,536 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <!-- header start -->
+
+    <header class="header">
+      <a
+        href="./index.html"
+        class="header__link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo Company Name Logo"
+        />
+      </a>
+
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a
+              href="./index.html"
+              class="header__nav-link is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="./index.html"
+              class="header__nav-link"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="./index.html"
+              class="header__nav-link"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="./index.html"
+              class="header__nav-link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="./index.html"
+              class="header__nav-link"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="./index.html"
+              class="header__nav-link"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="./index.html"
+              class="header__nav-link"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="./index.html"
+              class="header__nav-link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <!-- header ends  -->
+
+    <!-- cards starts -->
+    <main class="cards">
+      <!-- card #1 -->
+      <section
+        class="product-card"
+        data-qa="card"
```

Comment:
Each product card should have a unique `data-qa` attribute value to facilitate automated testing. Please ensure that each card has a distinct `data-qa` value, such as `data-qa="card-index"` where `index` is a unique number for each card.